tests/TestCase/ORM/MarshallerTest.php

```diff
@@ -2679,7 +2727,7 @@ public function testJoinDataWhiteList()
         $marshall = new Marshaller($this->articles);
         $result = $marshall->one($data, [
             'associated' => [
-                'Tags._joinData' => ['fieldList' => ['active', 'user']],
+                'Tags._joinData' => [$fields => ['active', 'user']],
```

Comment:
Could we keep the old and new test alongside and mark the old tests one deprecated (with version)?